labeler/has_pull_request.go

```diff
@@ -0,0 +1,47 @@
+package labeler
+
+import (
+	"log"
+	"regexp"
+	"strconv"
+
+	"github.com/google/go-github/github"
+	"github.com/parkr/auto-reply/ctx"
+)
+
+var fixesIssueMatcher = regexp.MustCompile(`Fixes #(\d+)`)
```

Comment:
I never wrote a line of Go. Did you take a look at existing tests as https://github.com/parkr/auto-reply/blob/bcf3d00d85f0495d2938b74a65884ea7c9d7e8c9/chlog/create_release_on_tag_test.go#L22-L35?